SlevomatCodingStandard/Sniffs/Classes/PropertyPerClassLimitSniff.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Classes;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SlevomatCodingStandard\Helpers\ClassHelper;
+use SlevomatCodingStandard\Helpers\SniffSettingsHelper;
+use function sprintf;
+use const T_CLASS;
+use const T_TRAIT;
+
+final class PropertyPerClassLimitSniff implements Sniff
+{
+
+	public const CODE_PROPERTY_PER_CLASS_LIMIT = 'PropertyPerClassLimit';
+
+	/** @var int */
+	public $maxCount = 10;
+
+	/**
+	 * @return array<int, (int|string)>
+	 */
+	public function register(): array
+	{
+		return [T_CLASS, T_TRAIT];
```

Comment:
By adding `T_ANON_CLASS` to this array, this sniff can also be used for anonymous classes. I don't think any extra changes are required (except for adding a few test cases maybe).